corehq/ex-submodules/phonelog/admin.py

```diff
@@ -20,8 +31,6 @@ class DeviceReportEntryAdmin(admin.ModelAdmin):
         'date',
         'username',
     ]
-    # postgres SQL count query takes long time on big tables
-    show_full_result_count = False
```

Comment:
Surprised that this didn't work.

Reply:
Yeah, I was surprised too - the [docs](https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#django.contrib.admin.ModelAdmin.show_full_result_count) say:
>  Set show_full_result_count to control whether the full count of objects should be displayed on a filtered admin page (e.g. 99 results (103 total)). If this option is set to False, a text like 99 results (Show all) is displayed instead.
> The default of show_full_result_count=True generates a query to perform a full count on the table which can be expensive if the table contains a large number of rows.

The problem was that if the admin page wasn't filtered, it'd still show the full count, and even filtered it would show _a_ count.  The filtered version of the report actually worked after adding that though.